# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1287366419 14400
#      Sun Oct 17 21:46:59 2010 -0400
# Node ID d4fc294a19e42cc9e75aa3bac11a9af1fd1d8f6f
# Parent  eab173e1e80c93e9332499312a67d5570356fcfe
include untabify docstring in the manual

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,12 @@
+2010-10-17  John W. Eaton  <jwe@octave.org>
+
+	* interpreter/strings.txi: Document untabify.
+
 2010-09-30  John W. Eaton  <jwe@octave.org>
 
 	* interpreter/contributors.in: Add Jordi Gutiérrez Hermoso to
 	the list.  Fix spelling of Søren Hauberg.
 
 2010-09-28  Rik  <octave@nomad.inbox5.com>
 
 	* doc/interpreter/Makefile.am, doc/interpreter/geometryimages.m,
diff --git a/doc/interpreter/strings.txi b/doc/interpreter/strings.txi
--- a/doc/interpreter/strings.txi
+++ b/doc/interpreter/strings.txi
@@ -443,16 +443,18 @@ general regular expressions, the followi
 @DOCSTRING(regexp)
 
 @DOCSTRING(regexpi)
 
 @DOCSTRING(regexprep)
 
 @DOCSTRING(regexptranslate)
 
+@DOCSTRING(untabify)
+
 @node String Conversions
 @section String Conversions
 
 Octave supports various kinds of conversions between strings and
 numbers.  As an example, it is possible to convert a string containing
 a hexadecimal number to a floating point number.
 
 @example
